src/trio/_core/_instrumentation.py

```diff
@@ -34,6 +35,7 @@ class Instruments(dict[str, dict[Instrument, None]]):
     __slots__ = ()
 
     def __init__(self, incoming: Sequence[Instrument]) -> None:
+        super().__init__({})
         self["_all"] = {}
```

Comment:
I guess this could be passed in `super().__init__({})` (and generally I suppose `self["..."] = ...` could become `self.data["..."] = ...`?)... That could be a follow up too.